internal/timeutil/location/loc_test.go

```diff
@@ -0,0 +1,261 @@
+				if !reflect.DeepEqual(got, time.UTC) {
+					return errors.Errorf("not equals. want: %v, but got: %v", time.UTC, got)
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns UTC location when loc is Utc",
+			loc:  "UTc",
+			checkFunc: func(got *time.Location) error {
+				if !reflect.DeepEqual(got, time.UTC) {
+					return errors.Errorf("not equals. want: %v, but got: %v", time.UTC, got)
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns GMT location when loc is GMT",
+			loc:  locationGMT,
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), locationGMT; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns GMT location when loc is Gmt",
+			loc:  "Gmt",
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), locationGMT; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns JST location when loc is JST",
+			loc:  locationJST,
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), locationJST; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns JST location when loc is Jst",
+			loc:  "Jst",
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), "Jst"; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns JST location when loc is Asia/Tokyo",
+			loc:  locationTokyo,
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), locationTokyo; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns JST location when loc is ASIA/Tokyo",
+			loc:  "ASIA/Tokyo",
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), "ASIA/Tokyo"; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "returns invalid location when loc is invalid",
+			loc:  "invalid",
+			checkFunc: func(got *time.Location) error {
+				if got == nil {
+					return errors.New("got is nil")
+				} else if got, want := got.String(), "invalid"; got != want {
+					return errors.Errorf("String() not equals. want: %v, but got: %v")
+				}
+				return nil
+			},
+		},
```

Comment:
- Could you add a test case `returns invalid location when loc is default value` ?